src/partition.rs

```diff
@@ -214,6 +214,16 @@ impl BlockSize {
 
     (offset_x, offset_y)
   }
+  
+  pub fn larger_than(self, other: BlockSize) -> bool {
+    (self.width() > other.width() && self.height() >= other.height()) ||
+    (self.width() >= other.width() && self.height() > other.height())
+  }
+
+  pub fn lte(self, other: BlockSize) -> bool {
```

Comment:
Rename this to larger_than_or_equal_to? "lte" could be interpreted as "less than or equal to"

Reply:
Indeed, thanks for pointing out. (I was also seeing that likely misleading). Done.